fvp/optee_sp_manifest.dts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2020, Arm Limited. All rights reserved.
+ * Copyright (c) 2022, Linaro Limited
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * This file is a Partition Manifest (PM) for a minimal Secure Partition (SP)
+ * that has additional optional properties defined.
+ *
+ */
+
+/dts-v1/;
+
+/ {
+	compatible = "arm,ffa-manifest-1.0";
+
+	/* Properties */
+	description = "op-tee";
+	ffa-version = <0x00010001>; /* 31:16 - Major, 15:0 - Minor */
+	uuid = <0xe0786148 0xe311f8e7 0x02005ebc 0x1bc5d5a5>;
+	id = <1>;
+	execution-ctx-count = <8>;
+	exception-level = <2>; /* S-EL1 */
+	execution-state = <0>; /* AARCH64 */
+	load-address = <0x6280000>;
+	mem-size = <0xd00000>; 	/* OP-TEE specific extension */
+	entrypoint-offset = <0x4000>;
+	xlat-granule = <0>; /* 4KiB */
+	boot-order = <0>;
+	messaging-method = <0x3>; /* Direct request/response supported. */
+	ns-interrupts-action = <1>; /* NS_ACTION_ME */
+	notification-support;
+	managed-exit;
+	run-time-model = <1>; /* SP pre-emptible. */
```

Comment:
run-time-model isn't consumed by hafnium.

Reply:
I wasn't aware of that, I think I copied it from TF-A FVP or from QEMU. I will drop it. I also noticed
notification-support and managed-exit are also not needed. I have dropped those 2 in my change request to
TF-A (https://review.trustedfirmware.org/c/TF-A/trusted-firmware-a/+/29047 https://review.trustedfirmware.org/c/TF-A/trusted-firmware-a/+/29041).

The idea is to get them updated or added in TF-A and drop them here when we move to the TF-A release containing those files so that only one copy can be maintained.